### PR TITLE
Fix users listing

### PR DESCRIPTION
**Pull request by DadImScared**

### CODE REVIEW

**Comment by DadImScared** [comment]
[Users listing](https://trello.com/c/hUJiBzkg/57-users-listing)

File: frontend/src/components/Users/InlineTextEdit.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-edit-dialog
+    :return-value.sync="item.notes"
+    lazy
+    large
+    @save="$listeners.save(item, editPath, item.notes)"
+    @cancel="$listeners.cancel"
```
Comment (anantakrishna) [inline frontend/src/components/Users/InlineTextEdit.vue]: Why `listeners` is here?

Comment (DadImScared) [reply frontend/src/components/Users/InlineTextEdit.vue]: The save and cancel functions are passed as events.

Comment (anantakrishna) [reply frontend/src/components/Users/InlineTextEdit.vue]: I mean, why it is called “listeners” in this context? I don't understand this.

**Comment by anantakrishna** [comment]
Oh, I got it now. I was thinking that it is somewhat related to the
listening please of our workflow... And I was wondering why it is mentioned
here, in the users list.

**Pull request merged**